src/libraries/System.Private.CoreLib/src/Internal/Console.Android.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
+        {
+            Interop.Logcat.AndroidLogPrint(Interop.Logcat.LogLevel.Debug, "DOTNET", s);
```

Comment:
Does __android_log_print support a null format being passed?  The existing use won't ever pass null; wondering if this should be `s ?? string.Empty`.

Reply:
I don't think it does, seems like it would be a NullPointerException https://developer.android.com/ndk/reference/group/logging#__android_log_print

I can add that to ensure we don't pass null. Out of curiosity, could `s` be null if we never change the method to accept a nullable parameter?